CONTRIBUTING.md

```diff
@@ -97,7 +97,10 @@ Before a pull request can be merged:
 * The contribution must follow the style guidelines of this project
 * Multiple commits should be used if the PR is complex and clarity can be improved, but they should still relate to a single topic
 * For code contributions, tests have to be added/modified to ensure the code works
-* There has to be at least one approval
+* The pull request must be approved by team members:
```

Comment:
I think we should clarify that this means members of this GitHub team: https://github.com/orgs/enterprise-contract/teams/devs

Reply:
Good idea. I updated the PR with that link and a little more clarification.